packages/slice-machine/lib/builders/CustomTypeBuilder/SliceZone/index.tsx

```diff
@@ -36,7 +35,7 @@ const mapAvailableAndSharedSlices = (
   }>(
     (acc, { key, value }) => {
       // Shared Slice
-      if (value.type === SlicesTypes.SharedSlice) {
+      if (value.type === "SharedSlice") {
```

Comment:
What triggered this change? Should this be our new standard? Is it checked by eslint/typescript?

Reply:
It is type-safe, yes. Having to use the constant all the time instead of the string literal was unnecessary indirection when TypeScript protects us in both cases anyway. It's not only the new standard, it's also the only way now because the library doesn't expose those constants anymore.